lib/workers/package/versions.js

```diff
@@ -223,3 +227,16 @@ function isPastLatest(npmDep, version) {
   logger.warn(`No dist-tags.latest for ${npmDep.name}`);
   return false;
 }
+
+function isAutomergeEnabled(automerge, type) {
+  /* eslint-disable no-fallthrough */
+  switch (type) {
+    case 'patch':
+      if (automerge === 'patch') return true;
+    case 'minor':
+      if (automerge === 'minor') return true;
+    default:
+      return automerge === 'any';
+  }
+  /* eslint-enable no-fallthrough */
```

Comment:
I prefer to write code verbosely rather than concisely if it means it's more understandable/maintainable in future.  So I would prefer that this section is refactored to avoid any `eslint-disable` and spell out the logic more obviously, e.g.

```
if (automerge === 'any') {
  return true;
}
if (automerge === 'minor' && type !== 'major') {
  return true;
}
if (automerge === 'patch' && type === 'patch') {
  return true;
}
return false;
```